src/transformers/models/led/modeling_led.py

```diff
@@ -1572,6 +1572,10 @@ class LEDSeq2SeqQuestionAnsweringModelOutput(ModelOutput):
             you can choose to directly pass an embedded representation. This is useful if you want more control over
             how to convert `input_ids` indices into associated vectors than the model's internal embedding lookup
             matrix.
+        inputs_embeds (`torch.FloatTensor` of shape `(batch_size, sequence_length, hidden_size)`, *optional*):
```

Comment:
They are actually in the paragraph above, just badly formatted. Could you remove them from there?

Reply:
@sgugger thanks for the review. I deleted that docstring and corrected the "badly formatted" one.